src/test/java/nextstep/ladder/domain/PlayerTest.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.exception.LadderException;
+import nextstep.ladder.exception.LadderExceptionCode;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class PlayerTest {
+
+    @Test
+    void nameIsEmpty_ThrowException() {
+        assertThatThrownBy(() -> new Player(null))
+                .isInstanceOf(LadderException.class);
+        assertThatThrownBy(() -> new Player(""))
+                .isInstanceOf(LadderException.class);
+    }
```

Comment:
테스트 전달 파라미터가 많아질 수록 코드가 계속 늘어나게 될 수 있는데요.
@ParameterizedTest를 활용해보는 건 어떨까요? 😄 
```suggestion
    @ParameterizedTest
    @NullAndEmptySource
    void nameIsEmpty_ThrowException(String name) {
        assertThatThrownBy(() -> new Player(name))
                .isInstanceOf(LadderException.class);
    }
```
아래 테스트 케이스도 동일합니다.